homeassistant/components/media_player/__init__.py

```diff
@@ -811,6 +816,10 @@ def entity_picture(self):
         url = self.media_image_url
 
         if url is None:
+            if self.media_image is not None:
+                url = 'media_image'
```

Comment:
A constant url will lead to constant md5 and the browser caching this image.

The md5 should run on the image itself, or part of it or a separate property.